### PR TITLE
doc: docker-compose version

### LINKED ISSUE #192: integration script start fails
**Problem:** 
The integration poc start script fails to start the docker services using docker-compose version: 1.25.0
The script reports error as below.
+ INT_BUILD=build
+ INT_START=start
+ INT_TESTS=tests
+ INT_LOGS=logs
+ INT_STOP=stop
+ DC=docker-compose
++ command -v docker-compose
+ '[' /usr/bin/docker-compose == '' ']'
+ '[' 1 -lt 1 ']'
+ '[' start == build ']'
+ '[' start == start ']'
+ start_containers
+ bash -c 'docker-compose down'
/usr/lib/python3/dist-packages/paramiko/transport.py:219: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,
ERROR: The Compose file './docker-compose.host.yml' is invalid because:
Unsupported config option for services.host-bmc: 'volumes_from'
Unsupported config option for services.host-cpu: 'volumes_from'

**Environment:**
Docker version: client: 20.10.12 server: 20.10.14
Docker-Compose version: 1.25.0
OS: Ubuntu 20.04.3 LTS

**Steps to reproduce:**
1. checkout opi-poc source
2. change to 'integration' directory 
3. run the command './scripts/integration.sh start'

### PR DESCRIPTION
**Pull request by glimchb**
Fixes #192

Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: integration/README.md [hunk 1]
```diff
@@ -6,7 +6,12 @@
 
 ## Prereqs
 
-Install docker-compose
+Install docker-compose <https://docs.docker.com/compose/install/>
+
+```bash
+ $ docker-compose -v
+docker-compose version 1.29.2, build unknown
```
Comment (mestery) [inline integration/README.md:13]: [1.29.2 is over a year old](https://github.com/docker/compose/releases/tag/1.29.2).. Can you also note this supported version which I am running:

```
vagrant@bullseye:/git/opi-poc/integration/opi-api$ docker compose version
Docker Compose version v2.6.0
vagrant@bullseye:/git/opi-poc/integration/opi-api$
```

[v2.6.0](https://github.com/docker/compose/releases/tag/v2.6.0)

**Review by mestery — CHANGES_REQUESTED** [review]
Lets make it clear we don't support docker compose earlier than 1.29.0, and we recommend using something recent like 2.6.0 or newer.

File: integration/README.md [hunk 2]
```diff
@@ -20,6 +27,14 @@ sudo systemctl enable podman.socket --now
 sudo setenforce 0
 ```
 
+### Prereqs - PIP
+
+one can also install latest cker-compose via PIP
```
Comment (seroyer) [inline integration/README.md:32]: Missing `do` on `cker-compose`

**Review by mestery — APPROVED** [review]
Stupendous!

File: integration/README.md (hunk 2 above)
Comment (glimchb) [reply integration/README.md]: thanks

**Review by seroyer — APPROVED** [review]

**Pull request merged**